client/src/components/App/App.scss

```diff
@@ -0,0 +1,115 @@
+.workbench-wrapper {
```

Comment:
The workbench-wrapper and workbench-content classes setup the sidebar and the content to the right, but should probably be renamed

Reply:
Going to rename this once I can track down the issue with the rest of the styling.